tests/test_main.py

```diff
@@ -41,7 +41,7 @@ def test_is_valid_position(self):
 
     def test_is_valid_position_invalid(self):
         invalid_positions = [
-            (2,3), (3,2), (3,3), (9,9), (-1,-1), 1, "something", False, (0,0,0)
+            (2,3), (3,2), (3,3), (9,9), (-1,-1), 1, "something", False, (0,0,0), ("st", 1)
         ]
         for position in invalid_positions:
             self.assertFalse(_position_is_valid(position))
```

Comment:
Is this because the test might fail for any number of the 10 possible options, but the only message you get is "True not False" or something like that, and that the entire test failed, you don't know which one?